core/src/main/java/com/scalar/db/storage/jdbc/JdbcAdmin.java

```diff
@@ -703,6 +709,24 @@ public Set<String> getNamespaceTableNames(String namespace) throws ExecutionExce
     }
   }
 
+  private Set<String> getNamespaceTableNamesInternal(Connection connection, String namespace)
```

Comment:
Ditto. Maybe, for consistency, we should rename this method to `getRawTableNames(String)` or `getInternalTableNames(String)`.